maven_plugin_config-master/pom.xml

```
@@ -14,7 +14,7 @@
   <parent>
     <groupId>org.eclipse.scout</groupId>
     <artifactId>maven_plugin_version-master</artifactId>
-    <version>26.1.0-SNAPSHOT</version>
+    <version>26.1.0-lto-SNAPSHOT</version>
```

Just as reminder: `lto-` tag can be removed

```suggestion
    <version>26.1.0-SNAPSHOT</version>
```